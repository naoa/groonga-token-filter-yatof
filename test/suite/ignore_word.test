register token_filters/yatof

table_create ignore_words TABLE_HASH_KEY ShortText
load --table ignore_words
[
{"_key": "and"}
]

tokenize TokenBigram "Hello and World" \
  --normalizer NormalizerAuto \
  --token_filters TokenFilterIgnoreWord

