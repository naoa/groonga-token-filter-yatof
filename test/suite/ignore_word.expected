register token_filters/yatof
[[0,0.0,0.0],true]
table_create #ignore_words TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
load --table #ignore_words
[
{"_key": "and"}
]
[[0,0.0,0.0],1]
tokenize TokenBigram "Hello and World"   --normalizer NormalizerAuto   --token_filters TokenFilterIgnoreWord
[[0,0.0,0.0],[{"value":"hello","position":0},{"value":"world","position":1}]]
